CakePHP/Tests/Commenting/DocBlockAlignmentUnitTest.php

```diff
@@ -16,8 +16,6 @@ public function getErrorList($testFile = '')
                 return [
                     2 => 1,
                     7 => 1,
-                    14 => 1,
-                    21 => 1,
```

Comment:
Yes, but it will also allow correctly-formatted doc comments that were previously failing this sniff to pass. Those removed lines should be re-added in a subsequent commit that correctly sniffs for **method** doc block alignment issues.

That would have been included in my commits, but I'm still wrapping my head around how codesniffer tests work. I'll take a crack at it, though.